tests/dummy/src/resources/user.ts

```diff
@@ -0,0 +1,7 @@
+import { Resource } from "../jsonapi-ts";
+
+export default class User extends Resource {
+  static attributes = {
```

Comment:
How are we declaring that a User has many posts or whatever relationship exists between these two resources?

Reply:
We don't have resource relationships yet :/

Reply:
Even if it's something temporary I think we should look at how we declare relations before we can tackle includes.

Reply:
Can we not have these kinds of discussions on GH? That is what Slack is for FFS